scripts/bundle.js

```diff
@@ -1,41 +1,62 @@
 #!/usr/bin/env node
+/* eslint-env node */
 import { build } from 'esbuild';
-
+import { promisify } from 'node:util';
+import { join } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { litCssPlugin } from 'esbuild-plugin-lit-css';
+import Glob from 'glob';
 import CleanCSS from 'clean-css';
 
-import { litCssPlugin } from 'esbuild-plugin-lit-css';
+const glob = promisify(Glob);
+
+const resolveDir = join(fileURLToPath(import.meta.url), '../..//elements');
```

Comment:
Should this be `'../../elements'` instead of  `'../..//elements'`?

Reply:
thanks fixed